products/jse_sub/behat.yml

```diff
@@ -0,0 +1,15 @@
+imports:
```

Comment:
@kmakitan, 
How about we hold off on this. I'd like to think about it some more.